app/presenters/linked_item_presenter.rb

```diff
@@ -44,4 +49,17 @@ def web_url(item)
 
     Plek.current.website_root + item.base_path
   end
+
+  def add_parents_recursively(parent)
+    json = {
+      content_id: parent.content_id,
+      title: parent.title,
+      base_path: parent.base_path
```

Comment:
Not sure about these 3 fields. Note that the frontend potentially needs more: https://github.com/alphagov/govuk-content-schemas/pull/279/commits/f2b26495258a990482d90b47c6633fae635f6618.

What's a cleaner way to get the "parent" representation of this content item?